instruqt-tracks/terraform-cloud-aws-v2/01-setup-our-environment/assignment.md

```diff
@@ -60,12 +60,12 @@ timelimit: 1800
     padding: 2px 10px 2px 5px;
     font-size: 14px;
     letter-spacing: 1.2px;
-    align-items: center;
```

Comment:
The presence of this line in this block is different between different versions of the track.  In AWS it's been removed but in Azure and GCP it's still there.  All tracks have `justify-content: center;` in this block.

Reply:
Thanks again, @omkensey . Added the right property for flex alignment.